UIDD/UIDD-Session2/stylesheet.css

```diff
@@ -0,0 +1,357 @@
+body{
+	font-style: aerial;
+}
+
+#topscreen {
+background-image: url("images/homebg.png");
+	background-size: 100% 100%;
+	background-repeat: no-repeat;
+}
+
+#topscreenlogo{
+	float: left;
+	text-align: center;
+	margin-top: 3%;
+}
+
+#googleplus{
+	height: 16px;
+	padding-bottom: 1%;
+}
+
+#instagram{
+	margin-top: 10px;
+}
+.inline{
+	margin: 2% 0% 0% 0%;
+}
+
+*{
+	margin: 0px;
+	padding: 0px;
+}
+
+.flex{
+	display: flex;
+	padding-bottom: .8%;
+}
+.center{
+	margin: auto; 
+	font-size: 135%;
+	padding-top: 2%;
+}
+
+nav a{
```

Comment:
adding styles directly to a tag  might cause issues when this tag will be used at other place.